app/api/api/views.py

```diff
@@ -1005,53 +1006,6 @@ class ScanReportConceptFilterViewSet(viewsets.ModelViewSet):
     }
 
 
-class ScanReportActiveConceptFilterViewSet(viewsets.ModelViewSet):
```

Comment:
Has this been superceded already? This is/was the function used by the auto-mapping functions to look across all Active SRs for reusable mappings.

Reply:
Yes the mapping reuse is completely database driven now - no API is used.